questcreation.md

```diff
@@ -1,42 +1,41 @@
 # Quests Creation
 
-### Terminology
+### **Terminology**
 
-- **Quest:** A series of steps, all connected together based on their topic.
-- **Step:** A small task inside a quest. Usually, a step will represent a single user interaction.
-- **Trigger:** An event in the system caused by the user. e.g., New chat message, PR opened, env var added to Heroku, etc.
-- **Action:** A reaction to a triggered event that causes the user’s state in the quest to change. e.g., advance to next step, approve/reject user’s PR, send a message from bot, etc.
+- **Step:** A task that usually represents a single user interaction.
```

Comment:
Logically, I don't think we should change the order between Quests and Steps. Quest is the higher level of hierarchy.

Reply:
Yes, but we explain what a quest is by defining it through the lower hierarchy. In terms of comprehension, they first need to understand what a step is.